### PR TITLE
image_inference

### PR DESCRIPTION
**Pull request by DmitriyBartoshik**

### CODE REVIEW

File: inference_images.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+ap = argparse.ArgumentParser()
+ap.add_argument("-i", "--input", required=True)
+ap.add_argument("-o", "--output", required=True)
+args = vars(ap.parse_args())
+
+input_image_dir = args["input"]
+output_image_dir = args["output"]
+
+model_dir = '/content/drive/MyDrive/FGVC6/models/final/'
```
Comment (gudwin) [inline inference_images.py]: Can we move this into a separate config?

File: inference_images.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+model_path = '/content/drive/MyDrive/FGVC6/models/final/mask_rcnn_fashion_0025.h5'
+
+label_names = ['shirt, blouse', 'top, t-shirt, sweatshirt', 'sweater', 'cardigan', 'jacket', 'vest', 'pants', 'shorts',
```
Comment (gudwin) [inline inference_images.py]: Can we move configuration into the separate config too?

File: inference_images.py [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+            y2, x2 = np.max(mask_pos, axis=1)
+            rois[m, :] = [y1, x1, y2, x2]
+    return masks, rois
+
+for filename in os.listdir(input_image_dir):
+    if filename.lower().endswith(('.png', '.jpg', '.jpeg', '.tiff', '.bmp', '.gif')):
+        image_path = os.path.join(input_image_dir, filename)
+        print(image_path)
+        img = io.imread(image_path)
+        # img = cv2.imread(image_path)
```
Comment (gudwin) [inline inference_images.py]: Please remove commented code

File: inference_images.py (hunk 1 above)
Comment (DmitriyBartoshik) [reply inference_images.py]: I can transfer it to the parent class

File: inference_images.py (hunk 2 above)
Comment (DmitriyBartoshik) [reply inference_images.py]: and it too

File: inference_images.py (hunk 3 above)
Comment (DmitriyBartoshik) [reply inference_images.py]: ok!

**Pull request merged**